roxie/ccd/ccdmain.cpp

```diff
@@ -1240,8 +1240,8 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
         clientCert.privateKey.set(topology->queryProp("@privateKeyFileName"));
         clientCert.passphrase.set(topology->queryProp("@passphrase"));
         useHardLink = topology->getPropBool("@useHardLink", false);
-        maxFileAge[false] = topology->getPropInt("@localFilesExpire", (unsigned) -1);
-        maxFileAge[true] = topology->getPropInt("@remoteFilesExpire", 60*60*1000);
+        maxFileAgeNS[false] = milliToNano(topology->getPropInt("@localFilesExpire", (unsigned) -1));
```

Comment:
This will not return (unsigned __int64)-1 for the default case - which could cause some issues in later changes.

Reply:
Will fix. Well spotted